provider/pkg/provider/diff_test.go

```diff
@@ -15,9 +15,11 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-type object = map[string]any
-type list = []any
-type expected = map[string]*pulumirpc.PropertyDiff
+type (
+	object   = map[string]any
```

Comment:
nit: should we omit `=` to align with how we define types elsewhere.

Reply:
Ah, sorry, should have realized these are type aliases to make the test assertions easier to work with. We should leave these as-is.